.github/workflows/trivy.yml

```diff
@@ -23,11 +23,13 @@ jobs:
           docker build --pull -t docker.io/securefederatedai/openfl:${{ github.sha }} -f openfl-docker/Dockerfile.base .
 
       - name: Run Trivy vulnerability scanner
-        uses: aquasecurity/trivy-action@0.24.0
+        uses: aquasecurity/trivy-action@0.28.0
         with:
           image-ref: 'docker.io/securefederatedai/openfl:${{ github.sha }}'
           format: 'sarif'
           output: 'trivy-results.sarif'
+        env:
+          TRIVY_DB_REPOSITORY: 'ghcr.io/aquasecurity/trivy-db,public.ecr.aws/aquasecurity/trivy-db'
```

Comment:
IF the order matters in terms of failure, then I think keeping ecr ahead of ghcr would yield lesser failures due to higher limit

Reply:
Agree, however, I put it ghcr.io/aquasecurity/trivy-db first so,  we get updates quickly after all public.ecr.aws/aquasecurity/trivy-db is a mirror and there could be some latencies!
Also, failover is super quick!